samples/subsys/shell/shell_module/src/main.c

```diff
@@ -204,6 +257,10 @@ SHELL_CMD_ARG_REGISTER(version, NULL, "Show kernel version", cmd_version, 1, 0);
 
 void main(void)
 {
+#if defined(CONFIG_SHELL_START_OBSCURED)
```

Comment:
you could use `if (IS_ENABLED(CONFIG_SHELL_START_OBSCURED))` and avoid multiple ifdefs. Using `SHELL_COND_CMD_ARG_REGISTER`.

Reply:
done